src/content/reference/react/useContext.md

```diff
@@ -82,6 +82,11 @@ function Form() {
 ```
 
 It doesn't matter how many layers of components there are between the provider and the `Button`. When a `Button` *anywhere* inside of `Form` calls `useContext(ThemeContext)`, it will receive `"dark"` as the value.
+<Note>
+
+[`use(context)`](/reference/react/use) is preferred over `useContext` when you need to access context inside conditionals (like if) or loops (like for), as it is more flexible than `useContext`.
+
+</Note>
```

Comment:
Let's move this to the top, after `<Intro>` and have it say:

```
Starting in React 19, you can use `use(Context)` instead of `useContext(Context)`, which allows you to read context in conditionals and loops. 
```